src/main/java/io/pinecone/configs/PineconeConnection.java

```diff
@@ -16,8 +16,8 @@
 import java.util.concurrent.TimeUnit;
 
 /**
- * Handles communication with a Pinecone service or router. One PineconeConnection can be shared
- * and used concurrently by multiple threads.
+ * The {@link PineconeConnection} class handles communication with a Pinecone service or router. One PineconeConnection
+ * can be shared and used concurrently by multiple threads.
```

Comment:
And same comment here, I think?

Reply:
https://github.com/pinecone-io/pinecone-java-client/pull/110/files#r1561727854